iam-login-service/src/main/webapp/resources/iam/apps/dashboard-app/components/clients/clientslist/clientslist.component.html

```diff
@@ -88,6 +88,7 @@
                                 <div>
                                     <a ui-sref="client({id: c.client_id})">
                                         {{c.client_name}}
+                                    <span ng-if="!c.active"> (Suspended)</span>
                                     </a>
```

Comment:
We can separate (Suspended) from the client link and add it as a colored label under the client UUID

Reply:
![image](https://github.com/indigo-iam/iam/assets/99975605/addc7a1d-818e-44ff-be50-9c8d8a4cf7e7)

Reply:
> ![image](https://private-user-images.githubusercontent.com/99975605/316458785-addc7a1d-818e-44ff-be50-9c8d8a4cf7e7.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTEzNjA3ODksIm5iZiI6MTcxMTM2MDQ4OSwicGF0aCI6Ii85OTk3NTYwNS8zMTY0NTg3ODUtYWRkYzdhMWQtODE4ZS00NGZmLWJlNTAtOWM4ZDhhNGNmN2U3LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAzMjUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMzI1VDA5NTQ0OVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWM2NWJhOTA0M2YzMDllZGVmMjNiMGNlMzkzYzgxNGUxOGU4MDBiM2Q4YTEyYjk4ZTI3MDI1ZDM4ZDk5ZjE0YzMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.Ocaw0uD_G4FEVCtO5kQm-JpwtsNAzQXqDnXroBWcx_Y)

I personally would change the style/colour of the label to differentiate it from scopes, for example:

![Screenshot from 2024-03-25 12-20-15](https://github.com/indigo-iam/iam/assets/85555840/32412800-a27d-4c43-8471-7fa61877a5cd)

What do you think?

Reply:
Not bad. This one looks prominent and distinct.

Reply:
![image](https://github.com/indigo-iam/iam/assets/99975605/8310e1ae-4279-4b5a-9064-fbd0e282ecba)

I have used `label-danger` from Bootstrap for the label.